calculator.html

```diff
@@ -11,15 +11,35 @@
 
 </head>
 <body>
+
+
+
+        <div class="navbar">
+      
+         <a class="home"><a href="" > Home </a>
+
+          <a class="accessories"><a href="#Accessories">Contacts</a>
+
+           <a class="laptops"><a href="#Laptops"> About</a>
+            
+          <a class="contact"><a href="#Contact"> Scientific </a>
+          
+          <a class="about"><a href=""> Logarithmic</a>
+       </div>
+
+
      <h1>
         <div class="jumbotron">
             <P> MY CALCULATOR </P>
         </div>
      </h1>
+
+
+        
        
         <div class="wrap">
-            <form name="cal">
-            <input type="text" name="display">
+        <form name="cal">
+            <input type="text" name="display" readonly>
```

Comment:
In the next commit you delete the `readonly` attribute; no need to add it here. Please remove it